docs/features/pytorch.md

```diff
@@ -0,0 +1,147 @@
+# Pytorch Integration Guide
+
+## Overview
+This guide explains how to integrate PyTorch with `pixi`, it supports multiple ways of installing PyTorch.
+
+- Install PyTorch using `conda-forge` Conda channel (Recommended)
+- Install PyTorch using `pypi`, using our `uv`'s integration. (Most versions available)
+- Install PyTorch using `pytorch` Conda channel (Legacy)
+
+With these options you can choose the best way to install PyTorch based on your requirements.
+
+## Installing from Conda-forge
+You can install PyTorch using the `conda-forge` channel.
+These are the community maintained builds of PyTorch.
```

Comment:
I'd specify these are the "conda-forge community" maintained builds.  The "pytorch community" maintains another set.